src/context/WalletProvider/NewWalletViews/sections/InstalledWalletsSection.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, Button, Flex, Image, Stack, Text as CText } from '@chakra-ui/react'
+import { uniqBy } from 'lodash'
+import type { EIP6963ProviderDetail } from 'mipd'
+import { useCallback, useMemo } from 'react'
+import { isMobile } from 'react-device-detect'
+import { Text } from 'components/Text'
+import type { KeyManager } from 'context/WalletProvider/KeyManager'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { isMobile as isMobileApp } from 'lib/globals'
+import { useMipdProviders } from 'lib/mipd'
+
+const MipdProviderSelectItem = ({
+  provider,
+  connect,
+  isSelected,
+  isDisabled,
+}: {
+  provider: EIP6963ProviderDetail
+  connect: (adapter: string) => void
+  isSelected: boolean
+  isDisabled: boolean
+}) => {
+  const handleConnect = useCallback(
+    () => connect(provider.info.rdns),
+    [connect, provider.info.rdns],
+  )
+
+  return (
+    <Box
+      as={Button}
+      key={provider.info.rdns}
+      variant='ghost'
+      px={2}
+      ml={-2}
+      py={6}
+      borderRadius='md'
+      width='full'
+      onClick={handleConnect}
+      bg={isSelected ? 'whiteAlpha.100' : undefined}
+      isDisabled={isDisabled}
+    >
+      <Flex alignItems='center' width='full'>
+        <Image src={provider.info.icon} boxSize='24px' mr={3} />
+        <CText>{provider.info.name}</CText>
+      </Flex>
+    </Box>
+  )
+}
+
+export const InstalledWalletsSection = ({
+  modalType,
+  isLoading,
+  onConnect,
+}: {
+  modalType: string | null
+  isLoading: boolean
+  onConnect: () => void
+}) => {
+  const { connect } = useWallet()
+  const detectedMipdProviders = useMipdProviders()
+
+  const supportedStaticProviders = useMemo(() => {
+    if (isMobileApp || isMobile) return []
+    // TODO(gomes): This says installed, so we only display... installed. Static ones (MM, Rabby, XDEFI) will either be displayed as their own section (EVM wallets/others), or
+    // not at all, TBD with product
```

Comment:
@reallybeard planning to tackle this in a follow-up, wat do with the static options here? Doesn't fit into "Installed" since they're not installed (i.e if they are installed, they *will* display).

https://www.figma.com/design/om2yDg0BjLNNSkJz82O30N/Wallet-Modal?node-id=2818-216&p=f&t=gn4sAJNgMVsG8qJk-0
<img width="458" alt="image" src="https://github.com/user-attachments/assets/0268d4d6-b6a9-400d-bd85-fa7fa559d91d" />


1. Do we still want to have Rabby, Ctrl (XDEFI) and MM live as static with this new flow?
2. If so, where should these live, under "More" or somewhere else?

<img width="450" alt="image" src="https://github.com/user-attachments/assets/6730a49a-9152-4bdc-8d83-d67ce04c5ebd" />